server/initializers/checker-after-init.ts

```diff
@@ -200,6 +201,12 @@ function checkTranscodingConfig () {
   }
 }
 
+function checkImportConfig () {
+  if (CONFIG.IMPORT.SYNCHRONIZATION.ENABLED && !CONFIG.IMPORT.VIDEOS.HTTP) {
+    throw new Error("You need to enable HTTP import to allow synchronization")
```

Comment:
Use single quote

Reply:
Alright. If single quotes should be preferred, do you mind if I later open a PR to add this eslint rule?
```
"quotes": "off",
"@typescript-eslint/quotes": ["error", "single", { "avoidEscape": true }]
```

With all the fixes.

Reply:
It's in my todo, it's weird it was not activated